jest.config.js

```diff
@@ -26,5 +26,22 @@ module.exports = {
 				'<rootDir>/packages/telestion-client-prop-types/src/**/*(*.)@(spec|test).[jt]s'
 			]
 		}
+		// {
+		// 	displayName: 'MockServer',
+		// 	globals: {
+		// 		"ts-jest": {
+		// 			tsconfig: 'packages/vertx-mock-server/tsconfig.json'
+		// 		}
+		// 	},
+		// 	preset: 'ts-jest',
+		// 	testEnvironment: 'node',
+		// 	collectCoverageFrom: [
+		// 		'<rootDir>/packages/vertx-mock-server/src/**'
+		// 	],
+		// 	coveragePathIgnorePatterns: [],
+		// 	testMatch: [
+		// 		'<rootDir>/packages/vertx-mock-server/src/**/*(*.)@(spec|test).[jt]s'
+		// 	]
```

Comment:
Could you please add a `// TODO:` item here for what to do before uncommenting this section (adding an explanation of why it's commented out in the process)?